tests/VirtoCommerce.SimpleExportImportModule.Tests/CsvPagedPriceDataSourceTests.cs

```diff
@@ -0,0 +1,259 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CsvHelper;
+using Moq;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.CatalogModule.Core.Model.Search;
+using VirtoCommerce.CatalogModule.Core.Search;
+using VirtoCommerce.SimpleExportImportModule.Data.Services;
+using Xunit;
+
+namespace VirtoCommerce.SimpleExportImportModule.Tests
+{
+    [Trait("Category", "CI")]
+    public class CsvPagedPriceDataSourceTests
+    {
+        private const string CsvHeader = "SKU;Min quantity;List price;Sale price";
+        private static readonly string[] CsvRecords = { "TestSku1;1;100;99", "TestSku2;1;10;9.99", ";10;9;" };
+
+        [Theory]
+        [MemberData(nameof(GetCsvWithAndWithoutHeader))]
+        public async Task GetTotalCount_Calculate_AndReturnTotalCount(string[] records, string header)
+        {
+            await using var stream = await GetStream(GetCsv(records, header));
```

Comment:
Format tests according to Microsoft recommendations https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices#arranging-your-tests

Reply:
Done